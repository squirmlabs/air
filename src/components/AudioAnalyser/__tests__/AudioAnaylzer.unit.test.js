import React from 'react';
import { AudioAnaylzer } from '../AudioAnaylzer';
import { shallow } from 'enzyme';

describe('AudioAnaylzer', () => {
  test('renders AudioAnaylzer component correctly', () => {
    const wrapper = shallow(<AudioAnaylzer id="HAuXJVI_bUs" />);
    expect(wrapper).toMatchSnapshot();
  });

  test('renders null if id in AudioAnaylzer component not specified', () => {
    const wrapper = shallow(<AudioAnaylzer />);
    expect(wrapper).toMatchSnapshot();
  });
});
