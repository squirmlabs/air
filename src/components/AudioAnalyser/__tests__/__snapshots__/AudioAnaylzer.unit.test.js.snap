// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AudioAnaylzer renders null if id in audio-analyzer component not specified 1`] = `""`;

exports[`AudioAnaylzer renders audio-analyzer component correctly 1`] = `
<div
  className="audio-analyzer-container"
>
  <div
    className="audio-analyzer"
  >
    <iframe
      allow="autoplay; encrypted-media"
      allowFullScreen={true}
      className="audio-analyzer-player"
      frameBorder="0"
      src="https://www.youtube.com/embed/HAuXJVI_bUs"
      title="audio-analyzer"
    />
  </div>
</div>
`;
